leaflet-map.js

```diff
@@ -0,0 +1,46 @@
+(function() {
+	'use strict';
+	kintone.events.on('app.record.detail.show', function(event) {
+		const record = event.record;
+		const address = record.Address.value;
+		const headerSpace = kintone.app.record.getHeaderMenuSpaceElement();
+		const kintoneAddress = [37.789808, -122.401767];
+		const name = record.Place.value;
+
+		const mapSpace = document.createElement('div');
+		mapSpace.style.height = '300px';
+		mapSpace.setAttribute('id', 'kintone');
```

Comment:
Avoid using the `kintone` as an id because `kintone` is the value that might be used as an id of the baked-in kintone element in the future. Use another value as an id instead.